targettests/Remote-Sim/pauli_word.cpp

```diff
@@ -9,7 +9,6 @@
 // REQUIRES: remote-sim
 
 // clang-format off
-// RUN: nvq++ %cpp_std --target remote-mqpu --remote-mqpu-auto-launch 1 %s -o %t && %t
```

Comment:
I would recommend leaving this library mode test enabled by reverting the change to this file. Many of our other tests in this directory have both modes enabled as that provides better test coverage.